<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportItem</class>
 <widget class="QWidget" name="ReportItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblDatetime">
       <property name="text">
        <string>Datetime</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lytMain">
     <item>
      <widget class="QGroupBox" name="gboxNotes">
       <property name="title">
        <string>Notes</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="txtNotes"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gboxAttachments">
       <property name="title">
        <string>Attachments</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="lytAttachments"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
